testCourse/courseInstances/Sp15/assessments/exam12-sequentialQuestions/infoAssessment.json

```diff
@@ -6,6 +6,7 @@
   "set": "Exam",
   "number": "12",
   "advanceScorePerc": 100,
+  "shareSourcePublicly": true,
```

Comment:
We are not relying on this for the automated testing, but its nice to have here for manual testing.